Podcast/SubscriptionCollectionViewCell.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  SubscriptionCollectionViewCell.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 3/6/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class SubscriptionCollectionViewCell: RecommendedSeriesCollectionViewCell {
```

Comment:
Because its literally the exact same cell except for the label text.. I had it as a completely different cell at first then I thought that was super redundant .. If they end up changing we can change it but I think for now it just adds to the codebase for no purpose